client/me/account/toggle-use-community-translator.tsx

```diff
@@ -33,11 +33,10 @@ function ToggleUseCommunityTranslator() {
 				label={ translate( 'Enable the in-page translator where available. {{a}}Learn more{{/a}}', {
 					components: {
 						a: (
-							<a
-								target="_blank"
-								rel="noopener noreferrer"
+							<ExternalLink
 								href="https://translate.wordpress.com/community-translator/"
 								onClick={ handleLearnMoreClick }
+								children={ null }
```

Comment:
Out of curiosity, is this necessary?

Reply:
It's not necessary - the code compiles and runs without it. Due to the substitution via the translate call, `children` is provided at runtime. Omitting it does confuse the in-editor typescript linter into thinking there's an error, so this was my way of making that less confusing.

Maybe less confusing to just let the ide display its wrongness though?

Reply:
I didn't see the IDE complaining 🤷‍♂️

Reply:
Build now fails with 

client/me/account/toggle-use-community-translator.tsx(36,9): error TS2741: Property 'children' is missing in type '{ href: string; onClick: () => void; }' but required in type 'Omit<WordPressComponentProps<ExternalLinkProps, "a", false>, "target">'..

I'll revert my last commit.